personal_access_tokens_test.go

```diff
@@ -237,3 +237,16 @@ func TestRevokePersonalAccessToken(t *testing.T) {
 		t.Errorf("PersonalAccessTokens.RevokePersonalAccessToken returned error: %v", err)
 	}
 }
+
+func TestRotatePersonalAccessToken(t *testing.T) {
+	mux, client := setup(t)
+
+	mux.HandleFunc("/api/v4/personal_access_tokens/0/rotate", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, http.MethodPost)
+	})
+
+	_, _, err := client.PersonalAccessTokens.RotatePersonalAccessToken(0)
+	if err != nil {
+		t.Errorf("PersonalAccessTokens.RotatePersonalAccessToken returned error: %v", err)
+	}
+}
```

Comment:
Can you place this method above `TestRevokePersonalAccessToken` so it matches the ordering in the docs?

Reply:
Fixed in new push.